visual-diff.js

```diff
@@ -284,19 +286,31 @@ class VisualDiff {
 						h1 { font-size: 1.2rem; font-weight: 400; margin: 24px 0; }
 						h2 { font-size: 0.9rem; font-weight: 400; margin: 30px 0 18px 0; }
 						a { color: #006fbf; }
+						input { transform: scale(1.5); }
+						label { font-size: 0.9rem; }
 						.compare { display: flex; }
 						.compare > div { margin: 0 18px; }
 						.compare > div:first-child { margin: 0 18px 0 0; }
 						.compare > div:last-child { margin: 0 0 0 18px; }
 						.label { display: flex; font-size: 0.7rem; margin-bottom: 6px; }
 						.meta { font-size: 0.7rem; margin-top: 24px; }
 						.meta > div { margin-bottom: 3px; }
+						[hide-success] [success] { display: none; }
 					</style>
 				</head>
 				<body>
-					<h1>Visual-Diff</h1>${diffHtml}
+					<h1>Visual-Diff</h1>
+					<input id="hideSuccesses" type="checkbox"></input>
+					<label for="hideSuccesses">Show Only Failed Tests</label>
+					${diffHtml}
 					${createMetaHtml()}
 				</body>
+				<script>
+					const checkbox = document.querySelector('input[type="checkbox"]');
+					checkbox.addEventListener('change', function() {
+						document.body.toggleAttribute('hide-success');
+					});
```

Comment:
Suggest using an `id` in case we add more checkboxes later, and using CSS classes instead of attributes on native elements.

```suggestion
					const checkbox = document.getElementById('hideSuccesses');
					checkbox.addEventListener('change', () => {
						document.body.classList.toggle('hide-success', checkbox.checked);
					});
```